app/containers/Avatar/useAvatarETag.ts

```
@@ -48,8 +48,8 @@ export const useAvatarETag = ({
 				}
 
 				if (record) {
-					const observable = record.observe() as Observable<TSubscriptionModel | TUserModel | TLoggedUserModel>;
-					subscription = observable.subscribe(r => {
+					const observable = record.observe() as unknown as Observable<TSubscriptionModel | TUserModel | TLoggedUserModel>;
+					subscription = observable.subscribe((r) => {
```

ESLint

[error] Replace `(r)` with `r`

(prettier/prettier)